src/App.js

```diff
@@ -1,24 +1,234 @@
+            />
+            <MiniCard
+              iconType="tracking"
+              title="Tracking"
+              description="Track and save your medical history and health data"
+            />
+          </div>
+        </div>
+        <div className="flex justify-center mt-[10vw] mb-[20vw] ">
+          <Button
+            bgColor="bg-white"
+            hoverColor="hover:bg-primary-blue hover:text-white"
+            textColor="text-primary-blue"
+            border="border-2 border-primary-blue "
+          >
+            Learn more
+          </Button>
+        </div>
+        <div className="grid lg:grid-cols-2 grid-cols-1">
+          <img
+            src={articleIllustration02}
+            className="w-full"
+            alt="second article"
+          />
+          <div className="lg:pl-40 lg:pr-20">
+            <TextArticle
+              title="Leading healthcare providers"
+              text="Trafalgar provides progressive, and affordable healthcare, accessible on mobile and online for everyone. To us, it’s not just work. We take pride in the solutions we deliver"
+              centerText={false}
+            />
+            <Button
+              bgColor="bg-white"
+              hoverColor="hover:bg-primary-blue hover:text-white"
+              textColor="text-primary-blue"
+              border="border-2 border-primary-blue "
+            >
+              Learn more
+            </Button>
+          </div>
+        </div>
+      </div>
+      <div className="relative w-screen">
+        <LandingIcon type="dotts" className="absolute right-4" />
+      </div>
+      <div className="mx-[10vw] mt-[24vh] relative">
+        <div className="grid lg:grid-cols-2 grid-cols-1 mb-[20vw]">
+          <div className="lg:pl-40 lg:pr-20">
+            <TextArticle
+              title="Download our mobile apps"
+              text="Our dedicated patient engagement app and 
+                    web portal allow you to access information instantaneously (no tedeous form, long calls, 
+                    or administrative hassle) and securely"
+              centerText={false}
+            />
+            <Button
+              bgColor="bg-white"
+              hoverColor="hover:bg-primary-blue hover:text-white"
+              textColor="text-primary-blue"
+              border="border-2 border-primary-blue lg:mb-0 mb-[10vh] "
+            >
+              Download
+              <LandingIcon type="downArrow" fill="#458FF6" />
+            </Button>
+          </div>
+          <img
+            src={articleIllustration03}
+            className="w-full"
+            alt="third article"
+          />
+        </div>
+        <div className="relative">
+          <LandingIcon type="dotts" className="absolute -left-2 bottom-24 " />
+          <Carousel items={items} />
+        </div>
+        <div className="relative mt-[8vw]">
```

Comment:
For readability, you could also abstract the sections into their own components, and the App.js file would just read as a list of sections:
`<Navbar />`
`<OurServices />`
`<Carrousel />`
`<LatestArticles />`
`<Footer />`

Reply:
(the components are just examples that dont reflect the reality)